lib/carmen/region.rb

```diff
@@ -90,7 +90,7 @@ def load_subregions_from_path(path, parent=nil)
     # and overlaying matching data (if it exists) from the overlay_path.
     def load_data_at_path(path)
       data_sets = Carmen.data_paths.map do |data_path|
-        if File.exists?(data_path + path)
+        if File.exist?(data_path + path)
```

Comment:
can we use `File.join` here?

Reply:
We'll end up calling .exist? anyways once we get the file join - join just creates a path :)